templates-list.md

```diff
@@ -0,0 +1,49 @@
+# List of Templates to Create
+
+The list below includes all the templates we as templateers plan to create. Feel free to provide feedback on whether any templates need to be removed or added to the list. The "P{number}" before each item indicates the priority of completing this template. We want to focus mostly or now on completing P1 templates, but if you have a special interest in one of the other templates feel free to work on that too. Feedback on the priorities is also welcome. Items without "P{number}" are just categories, not templates to be created.
+
+* P1 Reference
+	* P3 api-reference
+	* P2 Glossary
+	* P4 FAQ
+* *Manual*
+	* P2 User guide
+	* P3 Installation guide
+	* P3 Developers' guide
+	* P4 Administrators’ guide
+	* P2 Defining personas guide
+* *Task*
+	* P1 Tutorial
+	* P1 How-to
+		* P1 Quickstart
+			* P3 Api-quickstart
+* *Concept*
+	* P2 Business overview (About page, Landing page)
+	* P4 Technical overview
+	* P4 Case Study
+	* P4 White paper
+	* P2 Explanation
+* *Community*
+	* P0 Community definitions
+	* P1 README
+	* P2 Contributors' guide
+	* P2 Code of Conduct
+	* P3 Issue template and merge request template
+	* P3 Project Governance - RFC processes?
+	* P3 License (Recommendations on attribution)
+* *Guide*
+	* P3 Style guide (for code)
+	* P3 Style guide (for documentation)
+* P3 Release notes
+* P4 Logging
+* P4 Pull request template
+* P4 Bug reporting template
+* P4 Comments (structured comments you can grok and grep)
```

Comment:
**Question (non-blocking):** I'm not sure what this template would be? It looks like it's based on the Conventional Comments guide?